clang/include/clang/StaticAnalyzer/Core/PathSensitive/LoopWidening.h

```diff
@@ -27,7 +27,8 @@ namespace ento {
 /// by the loop body in any iteration.
 ProgramStateRef getWidenedLoopState(ProgramStateRef PrevState,
                                     const LocationContext *LCtx,
-                                    unsigned BlockCount, const Stmt *LoopStmt);
+                                    unsigned BlockCount, const Stmt *LoopStmt,
```

Comment:
Do we need both the Stmt and the ElemRef here?

Reply:
Not any more, will remove.